autode/bracket/imagepair.py

```diff
@@ -195,6 +197,30 @@ def _align_species(self) -> None:
         rotated_p_mat = np.dot(rot_mat, p_mat.T).T
         self._left_image.coordinates = rotated_p_mat
 
+    def initialise_trj(
+        self,
+        left_history_name: str = "left_history_save.zip",
+        right_history_name: str = "right_history_save.zip",
+    ) -> None:
+        """
+        Initialise the trajectory save files (history of coordinates on
+        left and right images)
```

Comment:
Because the methods write separate trajectories for the left and right side (reactant and product essentially) at the end. So they have to be kept separate. And also the left and right coordinates need to be accessed separately (and that's linked to the histories)